{"uuid":"c7823ce2-5ff4-4be9-9c0c-f903c9801923","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["b5f2f3d9-26ea-4c67-9045-ba8b562cd07a"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"84bf8d01-ffa6-4fac-a9f2-c7b69ca85227-attachment","type":"image/png"},{"name":"Myscreenshot","source":"634f482c-3bd5-442e-a314-a576d396edd0-attachment"}],"parameters":[],"start":1696885797483,"stop":1696885797541},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885797542,"stop":1696885797542},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885797544,"stop":1696885797544},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885797545,"stop":1696885797545}],"start":1696885797471,"stop":1696885797546}